test/distributed/optim/test_zero_redundancy_optimizer.py

```diff
@@ -179,42 +179,6 @@ def step(self):
         o.step()
         self.assertEqual(x, torch.tensor([0.9], device=DEVICE))
 
-    def test_local_state_dict(self):
```

Comment:
no more "local state dict" concept, a bit error prone and not elastic, there was no user that I know of for this in fairscale